Riot/Modules/LaunchLoading/LoadingAnimation/ElementView.swift

```diff
@@ -102,7 +102,7 @@ public class ElementView: UIView {
         path.layer.bounds = CGRect(x: 0, y: 0, width: 55.2, height: 55.2)
         path.layer.masksToBounds = false
         path.shapeLayer.fillRule = CAShapeLayerFillRule.evenOdd
-        path.shapeLayer.fillColor = UIColor(displayP3Red: 0.052, green: 0.743, blue: 0.543, alpha: 1).cgColor
+        path.shapeLayer.fillColor = UIColor(rgb: 0x0DBD8B).cgColor
```

Comment:
Note for code review: This file says not to edit directly as it was generated by Flow, although I wonder if we still have the original anywhere?